content/work/adobe.md

```diff
@@ -39,4 +88,67 @@ summary: |
 ---
 
 {% import 'quotes.macros.njk' as quotes %}
-{{ quotes.grid(press) }}
+{% import 'embed.macros.njk' as embed %}
+
+More detailed description of what users can do here?
+
+{{ quotes.grid(press | slice(2) | first ) }}
+
+## What Adobe Needed
+
+Adobe wanted to make their fonts more searchable, tagged with natural language terms like “organic,” “edgy,” and “calligraphic.” They envisioned a fun and meditative game — a bit of joy in a difficult day — that would let players pick the tags for each font, and give Adobe the tools to track the user input. Adobe described it as “a game for people who have feelings about fonts” and requested custom illustrations, animations, and sound design. As typography and game lovers ourselves, the OddBird team was excited to dive in!
+
+{{ embed.icon_block(planning, 'Planning & Discovery Phase') }}
+
+## Design and Development Phase
+
+### Illustrations / Web Graphics
+
+While part of the team began setting up the infrastructure of the game, the designers got to work on illustrations. Sarah Hyndman of [Type Tasting] provided initial sketches. In collaboration with the animator, we established a style — solid, filled SVGs — for easy layer grouping and relatively small file size. In Illustrator, we created unique graphics for ten font questions and five typographic superpowers. We used Asset Export to ensure that our gradients and other styles would translate well to SVG. A tight back-and-forth via Slack between designers, animators, and Adobe helped us moved swiftly through drafting to final designs in order to meet the project deadline.
+
+[type tasting]: https://www.typetasting.com/
+
+<figure>
+<img src="{{ site.images }}work/adobe/enter-here.png" class="extend-full" alt="" /><figcaption>The vintage Ferris wheel
+went through a sketch phase and a couple of drafts before
+the design was finalized.</figcaption>
+</figure>
```

Comment:
@mirisuzanne is the preference to include markup like this ... or a macro for figure like:

```
{{ embed.figure(
  data=image-pretend.src,
  caption=image-pretend.caption | md
  class='caption'
) }}
```
I see it used both ways but I didn't see if we documented a preference moving forward.

Reply:
I think we've been preferring the macros, but we didn't go through and update all the legacy markdown

Reply:
All of the images on this page now use the `{{ embed.figure(...) }}` macro